nzedb/Music.php

```diff
@@ -78,12 +83,12 @@ public function __construct(array $options = [])
 		$this->privkey = Settings::value('APIs..amazonprivkey');
 		$this->asstag = Settings::value('APIs..amazonassociatetag');
 		$result = Settings::value('..maxmusicprocessed');
-		$this->musicqty = empty($result) ? $result : 150;
+		$this->musicqty = $result === null ? $result : 150;
 		$result = Settings::value('..amazonsleep');
-		$this->sleeptime = empty($result) ? $result : 1000;
+		$this->sleeptime = $result === null ? $result : 1000;
 		$this->imgSavePath = nZEDb_COVERS . 'music' . DS;
 		$this->renamed = '';
-		if (Settings::value('..lookupmusic') == 2) {
+		if ((int) Settings::value('..lookupmusic') === 2) {
```

Comment:
I'm in two minds about this. In theory the setting could contain values other than numbers, so the casting to int might mess up a text value. I know currently it *should* only have numeric values, I'm kind of ok with it, but my OCD streak is screaming :-)